sql3/parser/parser_test.go

```diff
@@ -250,40 +250,6 @@ func TestParser_ParseTimeUnitConstraints(t *testing.T) {
 	})
 }
 
-func TestParser_ParseTimeQuantumConstraints(t *testing.T) {
```

Comment:
So we took this test out, because there's no longer time quantum constraints. We are instead parsing these some other way. Where do we test the new thing's parsing?

Reply:
It's implicit in the tests that exist - `timequantum` constraints still exist, but only on time quantum types, not set types.

Reply:
Oh - actually, this is not correct. test reverted to work properly.